CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Crash with `OT-SVG` fonts on Linux/BSD
 - Crash during text compose on old GNOME under Wayland
 - Mouse cursor staying hidden after window regains focus on macOS Ventura
+- Blurry fonts when changing padding size at runtime
```

Comment:
It's not about fonts in the end, but general issue.

Maybe something like that, but blurry fonts is fine.
```suggestion
- Terminal becomes inconsistent when changing only padding
```

Reply:
I mean fonts is the only thing we do. I think it's best to document what the user experiences, so they understand what bug was fixed.

We gain nothing by having overly detailed technical changelogs when the user then has no clue what it is about.